templates/user/show.html.twig

```diff
@@ -6,9 +6,13 @@
 
 {% block body %}
 	{% if user.avatar %}
-	<div class="profil-pic-section">
-		<img class="profil-pic" src="{{'/uploads/' ~ user.avatar }}"/>
-	</div>
+		<div class="profil-pic-section">
+			<img class="profil-pic" src="{{'/uploads/' ~ user.avatar }}"/>
+		</div>
+	{% else %}
+		<div class="profil-pic-section">
+			<img class="profil-pic-default" src="{{ asset('build/images/usericon.png') }}" alt="user-icon"/>
+		</div>
 	{% endif %}
```

Comment:
try this:

    <div class="profil-pic-section">
        <img class="profil-pic" src="{{ user.avatar ? '/uploads/' ~ user.avatar : asset('build/images/usericon.png') }}"/>
    </div>

Reply:
oh right, that will save some lines, thanks !

Reply:
i just realized i need the "profil-pic-default" class from the image after the esle condition, is it possible to do it in another way than a ternary ?